prime-router/docs/api/lookup-tables-openapi.yaml

```diff
@@ -0,0 +1,174 @@
+openapi: "3.0.2"
+info:
+  title: ReportStream Lookup Table Management API
+  description: Fetches and manipulates lookup tables in ReportStream
+  version: "1.0"
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://open.cdc.gov
+    email: usds@cdc.gov
+
+paths:
+  /api/lookuptables:
+    get:
+      description: Fetch the list lookup tables
+      security:
+        - OktaAuth: [ ANONYMOUS ]
+      parameters:
+        - name: showInactive
+          in: query
+          description: Set to true to include inactive tables
+          schema:
+            type: boolean
+      responses:
+        '200':
+          description: The list of tables with version information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TableVersionInfo'
+        '400':
+          description: if there was an error fetching the list of tables
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMsg'
+
+  /api/lookuptables/{tableName}:
```

Comment:
Given the behavior of `/api/lookuptables` and `/api/lookuptables/{tableName}/{tableVersion}` I'd expect this to return a list of versions for this specific table. Is the intent for this to return the `latest` version of the table or the `active` version of a table? Endpoints that are aliases for each might be needed.

Reply:
We only want to return the active table data.  I can see how we can benefit from a better endpoint here.

Reply:
I added /content.  Take a look again